### PR TITLE
style: adjust pill spacing

### PR DESCRIPTION
**Pull request by gusnaughton**
# Alaska Airlines Pull Request

Adjusts pill based on pixel pushing feedback given during auro-flight review. We are reducing the space between the IATA code and the pill to 4px, and increasing the min-width of the duration pill to 72px.

## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Review by erikalanghauser — APPROVED** [review]
72px width looks good to go.

File: src/style-flight-segment.scss [hunk 1]
```diff
@@ -48,5 +48,9 @@
 .iata {
   margin-top: var(--auro-size-sm);
   color: var(--auro-color-text-secondary-on-light);
-  padding-bottom: var(--auro-size-xs);
+  padding-bottom: var(--auro-size-xxs);
+}
+
+auro-badge {
+  min-width: 72px;
```
Comment (blackfalcon) [inline src/style-flight-segment.scss:55]: So, var(--auro-size-xxxl) | 4rem is a thing. I feel like this `72px` is a guess. So I would recommend going with the size that is already part of the grid.

Comment (gusnaughton) [reply src/style-flight-segment.scss:55]: @erikalanghauser thoughts?

Comment (erikalanghauser) [reply src/style-flight-segment.scss:55]: @gusnaughton that is fine, please send a screenshot of what it looks like set to -size-xxxl. Thanks!

**Review by erikalanghauser — APPROVED** [review]
👍🏻

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**